src/testrel.jl

```diff
@@ -164,6 +164,7 @@ Transaction Step used for `test_rel`
     results), then `broken` can be used to mark the tests as broken and prevent the test
     from failing.
   - `readonly`: If true, run the query as readonly
+  - `disable_corerel_deprecations::Bool`: control CoreRel deprecations
```

Comment:
Does the Step need a disable_corerel_deprecations flag? It seems more of a test_rel attribute.

Reply:
I made a cleanup commit